test/unit/cog_api/fake/roles_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule CogApi.Fake.RolesTest do
+  use CogApi.FakeCase
+
+  doctest CogApi.Fake.Roles
+
+  setup do
+    CogApi.Fake.Server.start_link
+    :ok
+  end
+
+  describe "role_create" do
+    it "returns the created role" do
+      params = %{"name" => "foobar"}
```

Comment:
How do you feel about extracting the name here? I don't have a strong opinion either way